editoast/src/views/timetable/import.rs

```diff
@@ -183,19 +194,21 @@ async fn import_item(
     pf_request.with_rolling_stocks(&mut vec![rolling_stock.clone()]);
     // List operational points uic needed for this import
     let ops_uic = ops_uic_from_path(&import_item.path);
+    let ops_id = ops_id_from_path(&import_item.path);
     // Retrieve operational points
-    let op_id_to_parts = match find_operation_points(&ops_uic, infra_id, conn).await? {
-        Ok(op_id_to_parts) => op_id_to_parts,
-        Err(err) => {
-            return Ok(import_item
-                .trains
-                .into_iter()
-                .map(|train: TimetableImportTrain| (train.name, err.clone()))
-                .collect());
-        }
-    };
+    let (op_uic_to_parts, op_id_to_parts) =
```

Comment:
Maybe it would be better to manipulate directly an object like this, what do you think ?
```
op_to_parts= {
   from_uic: HashMap<i64, Vec<OperationalPointPart>>,
   from_id: HashMap<String, Vec<OperationalPointPart>>,
}
```
```suggestion
    let op_to_parts =
```